groovy-language-server/src/test/java/com/palantir/groovylanguageserver/GroovycWrapperTest.java

```diff
@@ -407,4 +982,24 @@ private File addFileToFolder(File parent, String filename, String contents) thro
         return file;
     }
 
+    private static SymbolInformation createSymbolInformation(String name, String uri, SymbolKind kind, int startLine,
```

Comment:
the number of args is making this unreadable -- consider a helper with the same signature as in `GroovycWrapper` where you pass in a Location directly